src/composables/multisigAccounts.ts

```diff
@@ -108,40 +122,41 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
               gaAccountId ? sdk.balance(gaAccountId) : 0,
             ]);
 
-            const consensus = consensusResult.decodedResult as any;
+            const decodedConsensus = consensusResult.decodedResult;
+            const txHash = decodedConsensus.tx_hash as Uint8Array;
+            const consensus: IMultisigConsensus = camelCaseKeysDeep(decodedConsensus);
 
-            if (consensus.tx_hash) {
-              consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
-            }
-            consensus.expiration_height = Number(consensus.expiration_height);
-            consensus.confirmations_required = Number(consensus.confirmations_required);
-            consensus.totalConfirmations = Number(consensus.confirmed_by.length);
+            consensus.expirationHeight = Number(consensus.expirationHeight);
+            consensus.confirmationsRequired = Number(consensus.confirmationsRequired);
 
             const consensusLabel = (
-              `${consensus?.confirmed_by?.length}/${consensus.confirmations_required} ${i18n.t('common.of')} ${signers.decodedResult?.length}`
+              `${consensus?.confirmedBy?.length}/${consensus.confirmationsRequired} ${i18n.t('common.of')} ${signers.decodedResult?.length}`
             );
 
             return {
-              ...camelcaseKeysDeep(consensus),
-              ...otherMultisig,
-              nonce: Number(nonce.decodedResult),
-              signers: signers.decodedResult,
+              ...consensus,
+              ...otherMultisigData,
               consensusLabel,
               contractId,
+              gaAccountId,
+              nonce: Number(nonce.decodedResult),
+              signers: signers.decodedResult,
               balance: convertToken(balance, -MAGNITUDE),
-              address: gaAccountId,
-              multisigAccountId: gaAccountId,
-              hasPendingTransaction: consensus.tx_hash && !consensus.expired,
+              refusedBy: [],
```

Comment:
```suggestion
```
This is one of the consensus fields which is being rewritten.

Reply:
OK, removed.